css/style.css

```
@@ -40,30 +47,44 @@ img {
 ul {
   list-style: none;
 }
-
+.center{
+  width: 1160;
+  margin: 0 auto;
+  padding: 0 20px;
+  list-style: none;
+}
 .main-navigation {
+  display: grid;
+  grid-template-columns: max-content 1fr 2fr max-content;
```

```suggestion
  grid-template-columns: max-content 1fr max-content;
```
В навигации три колонки, а не четыре.